modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb

```diff
@@ -8,7 +8,7 @@ class ClaimBlueprint < Blueprinter::Base
         field :claim_id
```

Comment:
Similar to the comment over on the JSON schema, but I've been trying to keep attributes listed in alphabetical order to make it easier to find things quickly.  Can we continue alphabetizing things here as well?  😄 🙏